### ISSUE TITLE
inline code does not have parent

### ISSUE THREAD

**Issue by dingus9** [codejamninja/sphinx-markdown-parser#2]
For inline code blocks marked with \`my code here\` I am getting this error

Sphinx error
```
# Sphinx version: 2.1.2
# Python version: 3.7.3 (CPython)
# Docutils version: 0.14
# Jinja2 version: 2.10.1
# Last messages:
#   Running Sphinx v2.1.2
#   building [mo]: targets for 0 po files that are out of date
#   building [html]: targets for 12 source files that are out of date
#   updating environment:
#   12 added, 0 changed, 0 removed
#   reading sources... [  8%] faas-functions
#   reading sources... [ 16%] index
#   reading sources... [ 25%] piperci-gman/README
# Loaded extensions:
#   sphinx.ext.mathjax (2.1.2) from /Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/ext/mathjax.py
#   sphinxcontrib.applehelp (1.0.1) from /Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinxcontrib/applehelp/__init__.py
#   sphinxcontrib.devhelp (1.0.1) from /Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinxcontrib/devhelp/__init__.py
#   sphinxcontrib.htmlhelp (1.0.2) from /Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinxcontrib/htmlhelp/__init__.py
#   sphinxcontrib.serializinghtml (1.1.3) from /Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinxcontrib/serializinghtml/__init__.py
#   sphinxcontrib.qthelp (1.0.2) from /Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinxcontrib/qthelp/__init__.py
#   alabaster (0.7.12) from /Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/alabaster/__init__.py
#   sphinx.ext.autodoc (2.1.2) from /Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/ext/autodoc/__init__.py
#   sphinxcontrib.autoprogram (unknown version) from /Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinxcontrib/autoprogram.py
#   sphinx_markdown_parser (0.1.0) from /Users/nickshobe/Projects/sphinx-markdown-parser/sphinx_markdown_parser/__init__.py
Traceback (most recent call last):
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/cmd/build.py", line 284, in build_main
    app.build(args.force_all, filenames)
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/application.py", line 345, in build
    self.builder.build_update()
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 319, in build_update
    len(to_build))
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 332, in build
    updated_docnames = set(self.read())
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 438, in read
    self._read_serial(docnames)
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 460, in _read_serial
    self.read_doc(docname)
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 504, in read_doc
    doctree = read_doc(self.app, self.env, self.env.doc2path(docname))
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/io.py", line 325, in read_doc
    pub.publish()
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/docutils/core.py", line 218, in publish
    self.apply_transforms()
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/docutils/core.py", line 199, in apply_transforms
    self.document.transformer.apply_transforms()
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/transforms/__init__.py", line 90, in apply_transforms
    super().apply_transforms()
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/docutils/transforms/__init__.py", line 171, in apply_transforms
    transform.apply(**kwargs)
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/transforms/__init__.py", line 340, in apply
    super().apply()
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/docutils/transforms/universal.py", line 307, in apply
    for txtnode, newtext in zip(txtnodes, teacher):
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/docutils/utils/smartquotes.py", line 570, in educate_tokens
    for (ttype, text) in text_tokens:
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/transforms/__init__.py", line 376, in get_tokens
    notsmartquotable = not is_smartquotable(txtnode)
  File "/Users/nickshobe/.virtualenvs/piperci-docs/lib/python3.7/site-packages/sphinx/util/nodes.py", line 503, in is_smartquotable
    elif node.parent.get('support_smartquotes', None) is False:
AttributeError: 'NoneType' object has no attribute 'get'
```

## example markdown causing this issue

```
# Piperci GMan

Some description

## Run dev server

test text `python test.py`
```

I've identified this issue exists as of commit 0056b9c89036db764926d2c9cd6e4d910751d3dc

**Comment by clayrisser** [comment]
Ok, thanks. I will look into it. Feel free to submit a pull request if you find the solution to the problem.

**Comment by dingus9** [comment]
I poked at it a little in pdb... but I'm not sure where the parent element is being lost

**Comment by clayrisser** [comment]
@dingus9 any suggestions or leads?

**Comment by clayrisser** [comment]
I fixed this in `v0.1.3`

**Comment by dingus9** [comment]
great I'll likely take a look at some point in the near future
